node/src/types.ts

```diff
@@ -480,12 +461,16 @@ export type Method =
   | 'build'
   | 'execute'
 
-export const DOCUMENT_FORMATS: Record<string, DocumentFormat> = {
+export const FORMATS: Record<string, Format> = {
```

Comment:
Should these have the `preview: boolean` attributes?

Reply:
Yes, it seems I may have not pushed that commit :/